### PR TITLE
Pull request for Initial publication of Average Anxiety Rating dataset

### PR DESCRIPTION
**Pull request by bailed22**
Transformation include:
1. wrangling.py
2. config.json
3. output.csv

### CODE REVIEW

File: anxiety/anxiety.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "keywords": [
+        "anxiety",
+        "average-rating",
+        "happiness-score",
+        "subnational",
+        "united-kingdom"
+    ],
+    "publisher": "https://www.gov.uk/government/organisations/office-for-national-statistics",
+    "themes": "https://www.ons.gov.uk/employmentandlabourmarket/peopleinwork",
```
Comment (sazzadur-ons) [inline anxiety/anxiety.json:16]: Anxiety should be related to health and wellbeing. The relevant theme is "https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/healthandwellbeing"

**Review by sazzadur-ons — COMMENTED** [review]
Commented for keywords and theme

File: anxiety/anxiety.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "title": "Anxiety",
+    "creator": "https://www.gov.uk/government/organisations/office-for-national-statistics",
+    "summary": "Average anxiety rating",
+    "description": "The average rating of those that felt anxious yesterday.  \nThe levelling up category is: Spreading opportunity and improving public services.  \nThe datasets covers united kingdom.\nOn 31 October 2022, alongside the publication of the latest personal well-being estimates for the period between April 2021 to March 2022,\npreviously published estimates for the periods between April 2011 and March 2019 have been updated due to changes in guidance on data suppression,\nwhere estimates are no longer required to be suppressed solely based on their co-efficient of variation (CV) values.",
+    "license": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "keywords": [
+        "anxiety",
```
[the remark this answers is not in the document]
Comment (bailed22) [reply anxiety/anxiety.json:9]: Noted. Correction implemented as requested

**Review by sazzadur-ons — COMMENTED** [review]
All are fine

File: anxiety/anxiety.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                    "from_existing": "https://purl.org/csv-cubed/resources/measures#count"
+                }
+            ]
+        },
+        "Observation": {
+            "type": "observations",
+            "unit": {
+                "label": "Score out of 10",
+                "from_existing": "http://qudt.org/vocab/unit/UNITLESS",
+                "description": "happiness out of 10"
```
Comment (sazzadur-ons) [inline anxiety/anxiety.json:50]: "description": "Happiness score out of 10"

**Pull request merged**